src/actions/users/users.js

```diff
@@ -69,12 +69,20 @@ export function getUser(id) {
  * @param {string} query Query
  * @returns {Object} List users action
  */
-export function listUsers(query) {
+export function listUsers(query, groups_filter) {
+  groups_filter = groups_filter || [];
+  let path = '/@users';
+  if (query) {
+    path += `?query=${query}`;
+    if (groups_filter.length > 0) {
+      path += `&groups_filter=${groups_filter.join(',')}`;
```

Comment:
@ksuess I think we should also have to change this `groups_filter` as per https://github.com/plone/plone.restapi/commit/bc42a349bb7740d7a61155d4f0c50c92a66c5c9f 🤔

Reply:
Yes of course, thank you @nileshgulia1